person.rb

```diff
@@ -19,6 +20,11 @@ def can_use_services?
     false
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
+
```

Comment:
- [ ] it would be better to add rental with its class like the below example here and in book class and in the rental class add a rental to books and person in the `initialize` method that way you will achieve the belongs to relationships
```ruby
// Book and Person class
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
// Rental class
  def initialize(date, book, person)
    @book = book
    *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
  end



```